src/truffle.js

```diff
@@ -3,6 +3,11 @@ require('babel-register')
 
 module.exports = {
   networks: {
+    ganache:{
```

Comment:
I think this file can be removed when there is a `truffle-config.js`

Reply:
Agreed :+1: